package.json

```diff
@@ -28,23 +21,24 @@
 	},
 	"lint-staged": {
 		"dotcom-rendering/**/*.md": [
-			"yarn workspace @guardian/dotcom-rendering createtoc",
-			"prettier --write --cache"
+			"pnpm --filter @guardian/dotcom-rendering createtoc",
+			"prettier --write"
 		],
-		"*": "prettier --ignore-unknown --write --cache"
+		"*": "prettier --ignore-unknown --write"
```

Comment:
Why is the `--cache` gone?

Reply:
i'm not what it offers here - we know the file has changed, so i don't think we will never want to skip formatting in this instance?

Reply:
though admittedly it's not pnpm-related and could be moved to a separate change